prow/plank/controller.go

```diff
@@ -255,10 +255,11 @@ func (c *Controller) syncJenkinsJob(pj kube.ProwJob, reports chan<- kube.ProwJob
 		pj.Status.State = kube.PendingState
 		c.incrementNumPendingJobs(pj.Spec.Job)
 		br := jenkins.BuildRequest{
-			JobName: pj.Spec.Job,
-			Refs:    pj.Spec.Refs.String(),
-			BaseRef: pj.Spec.Refs.BaseRef,
-			BaseSHA: pj.Spec.Refs.BaseSHA,
+			JobName:      pj.Spec.Job,
+			Refs:         pj.Spec.Refs.String(),
+			BaseRef:      pj.Spec.Refs.BaseRef,
+			BaseSHA:      pj.Spec.Refs.BaseSHA,
+			MetadataPath: c.ca.Config().Plank.BaseMetadataPath + pj.Status.UUID + "/",
```

Comment:
I wanted to allow users to have BaseMetadataPath = "" since that produces `<UUID>/` which you could still use. I don't think setting it will cause problems. Hopefully in the near future :tm: we can remove jenkins entirely.